mmv1/third_party/terraform/utils/provider.go.erb

```diff
@@ -482,17 +482,21 @@ end     # products.each do
 				"google_runtimeconfig_config":                  ResourceRuntimeconfigConfig(),
 				"google_runtimeconfig_variable":                ResourceRuntimeconfigVariable(),
 				<% end -%>
-				"google_service_account":                       ResourceGoogleServiceAccount(),
```

Comment:
I think these need to stay capitalized. We'll want to capitalize the Sharedflow ones as well. Sorry, this change happened within the time the PR was opened.